watchdog/map.jinja

```diff
@@ -11,10 +11,4 @@
         'parameter': {}
       }
     },
-    'QEMU': {
```

Comment:
as I said in previous review, instead of QEMU, put here `default`, then you'll have correct defaults on non-specific platforms and you will not need things like `|default('softdog')` bellow.

Reply:
Fixed...